app/controllers/api/cocktails_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Api::CocktailsController < ApplicationController
+  def index
+    response = RestClient.get(
```

Comment:
You should check incoming request has query params before making this call

Reply:
Fixed